build/build-plugins.js

```diff
@@ -8,6 +8,7 @@
 const rollup  = require('rollup')
 const path    = require('path')
 const babel   = require('rollup-plugin-babel')
+const banner  = require(path.resolve(__dirname, './banner.js'))
```

Comment:
Not a big fan of using `path.resolve` here because `require('./banner.js')` will works fine 🤔

Reply:
True, let's change it